feature-utils/poly-analysis/package.json

```diff
@@ -0,0 +1,12 @@
+{
+  "name": "@polypoly-eu/poly-analysis",
+  "main": "src/index.js",
+  "private": true,
+  "type": "module",
+  "scripts": {
+    "build": "rollup -c rollup.config.js"
```

Comment:
Think we can worry about that later - as long as the tests that cover this code are still in _facebookImport_, they're still running. But yeah, I agree that we should extract (generic) tests into this package.

Reply:
WIll put this in a ticket for a separate PR! So in the end this will have tests in both the package and the feature. Where the generic tests will be in the package and the feature specific tests will be in the feature?